packages/docusaurus-theme-classic/src/theme/DocSidebar/index.tsx

```diff
@@ -61,6 +66,8 @@ function DocSidebarItemCategory({
     }
     return isActive ? false : item.collapsed;
   });
+  const [goToLink, setGoToLink] = useState(false);
+  const [intitalHref, setIntitalHref] = useState('');
```

Comment:
typo

Reply:
+ I don't really understand why we need these 2 new states to implement this feature. 
We are just linking to an URL, why state is needed?

Reply:
Thank you for the swift response! 

Yes I agree I removed `goToLink` and now looking in renaming initialLink